### PR TITLE
Update `pineappl 0.8.2`

### PR DESCRIPTION
**Pull request by giacomomagni**
Update `pineappl` to version  `0.8.2`

### CODE REVIEW

**Review by Radonirinaunimi — APPROVED** [review]

**Comment by giacomomagni** [comment]
do we want to change the other occurrences of `convolute` in `convolve` before merging ?

**Comment by Radonirinaunimi** [comment]
> do we want to change the other occurrences of `convolute` in `convolve` before merging ?

I'd say no. We should just change the instances that are affected by pineappl. The pineappl renaming was already too much 🙈

**Comment by felixhekhorn** [comment]
> do we want to change the other occurrences of `convolute` in `convolve` before merging ?

I'd say yes :innocent: because the verb is just wrong (well non of us is a native speaker ...) - so you can have the decisive vote @giacomomagni if you wish

**Comment by giacomomagni** [comment]
Yes the math operation is `convolve` so better fix it.

**Comment by Radonirinaunimi** [comment]
Sure, but I don't think we should get into the habits of breaking APIs just for that sake. If we do so I am pretty sure that there will be many things that we could break. Here it is completely harmless because these things are not exposed, but still...

**Comment by giacomomagni** [comment]
I totally agree, but here we are only changing internal functions and docs, so it's actually a fix.

**Comment by giacomomagni** [comment]
shall I merge?

File: docs/source/implementation/scale-variations.rst [hunk 1]
```diff
@@ -58,15 +58,15 @@ scale variation formula <theory/scale-variations:Factorization>`.
...
 of extra numerical convolutions to a single one.
 
 In this case we are avoiding the disadvantages of the previous section, only
 partially paying for giving up on of the advantages: these convolutions would
 involve only splitting functions, and never the coefficient functions'
 expressions, and for this same reason they are also considerably easier.
 
-Applying convoluted kernels
+Applying convolved kernels
 ---------------------------
```
Comment (felixhekhorn) [inline docs/source/implementation/scale-variations.rst:70]:
```suggestion
--------------------------
```

File: extras/notes/apfel/APFEL_functions/DIS.md [hunk 2]
```diff
@@ -31,4 +31,4 @@
...
+CacheStructureFunctionsAPFEL.f, ComputeChargesDIS.f, ComputeDISOperators.f, ComputeStructureFunctionsAPFEL.f, ConvolveEvolutionWithDISOperators.f, ConvolvePDFsWithDISOperators.f, EnableDampingFONLL.f, EnableDynamicalScaleVariations.f, EnableIntrinsicCharm.f, EnableSFNLOQEDCorrections.f, EnableTargetMassCorrections.f, ExcludeScaleVariation.f, ExternalDISOperator.f, F2bottom.f, F2charm.f, F2light.f, F2top.f, F2total.f, F3bottom.f, F3charm.f, F3light.f, F3top.f, F3total.f, FKObservables.f, FKSimulator.f, FLbottom.f, FLcharm.f, FLlight.f, FLtop.f, FLtotal.f, GetCKM.f, GetFKObservable.f, GetGFermi.f, GetProtonMass.f, GetSIATotalCrossSection.f, GetSin2ThetaW.f, GetWMass.f, GetZMass.f, hqcoef.f, IncludeIntrinsicCharm.f, IncludeNLOQEDCorrections.f, IncludeScaleVariation.f, initIntegralsDIS.f, initIntegralsDISRes.f, initIntegralsSIA.f, integrandsDIS.f, integrandsIC.f, integrandsSIA.f, JoinDISOperators.f, LHAPDFgridStructureFunctions.f, MassiveCoefficientFunctions.f, MassiveZeroCoefficientFunctions.f, RSLintegralsDIS.f, RSLintegralsSIA.f, SelectCharge.f, SetFKObservable.f, StructureFunctionxQ.f, ZeroMassCoefficientFunctions.f
```
Comment (felixhekhorn) [inline extras/notes/apfel/APFEL_functions/DIS.md:34]:
```suggestion
CacheStructureFunctionsAPFEL.f, ComputeChargesDIS.f, ComputeDISOperators.f, ComputeStructureFunctionsAPFEL.f, ConvoluteEvolutionWithDISOperators.f, ConvolutePDFsWithDISOperators.f, EnableDampingFONLL.f, EnableDynamicalScaleVariations.f, EnableIntrinsicCharm.f, EnableSFNLOQEDCorrections.f, EnableTargetMassCorrections.f, ExcludeScaleVariation.f, ExternalDISOperator.f, F2bottom.f, F2charm.f, F2light.f, F2top.f, F2total.f, F3bottom.f, F3charm.f, F3light.f, F3top.f, F3total.f, FKObservables.f, FKSimulator.f, FLbottom.f, FLcharm.f, FLlight.f, FLtop.f, FLtotal.f, GetCKM.f, GetFKObservable.f, GetGFermi.f, GetProtonMass.f, GetSIATotalCrossSection.f, GetSin2ThetaW.f, GetWMass.f, GetZMass.f, hqcoef.f, IncludeIntrinsicCharm.f, IncludeNLOQEDCorrections.f, IncludeScaleVariation.f, initIntegralsDIS.f, initIntegralsDISRes.f, initIntegralsSIA.f, integrandsDIS.f, integrandsIC.f, integrandsSIA.f, JoinDISOperators.f, LHAPDFgridStructureFunctions.f, MassiveCoefficientFunctions.f, MassiveZeroCoefficientFunctions.f, RSLintegralsDIS.f, RSLintegralsSIA.f, SelectCharge.f, SetFKObservable.f, StructureFunctionxQ.f, ZeroMassCoefficientFunctions.f
```

File: extras/notes/apfel/TMC.md [hunk 3]
```diff
@@ -8,7 +8,7 @@ As in the docs pdf (`dis.pdf`)  aux variables are defined (`xi`, `rho` that is
 called `rhop` in the code) directly in `F2light.f`
 
 ## Integration
-Source: `ConvolutePDFwithDISOperators.f` for `I2`
+Source: `ConvolvePDFwithDISOperators.f` for `I2`
```
Comment (felixhekhorn) [inline extras/notes/apfel/TMC.md:11]:
```suggestion
Source: `ConvolutePDFwithDISOperators.f` for `I2`
```

**Review by felixhekhorn — APPROVED** [review]

**Comment by felixhekhorn** [comment]
> Sure, but I don't think we should get into the habits of breaking APIs just for that sake. If we do so I am pretty sure that there will be many things that we could break. Here it is completely harmless because these things are not exposed, but still...

on general grounds I agree - however, on eko we are facing the opposite: there are PRs which implement new good features, but are breaking and thus get delayed forever

**Pull request merged**